lib/rdkafka/callbacks.rb

```diff
@@ -157,6 +157,8 @@ def self.process_create_topic(event_ptr)
           create_topic_handle[:error_string] = create_topic_results[0].error_string
           create_topic_handle[:result_name] = create_topic_results[0].result_name
           create_topic_handle[:pending] = false
+
+          create_topic_handle.broadcast
```

Comment:
can we name it "unlock" or something similar? broadcast seems outside of our domain